jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/SysinfoCpuTime.java

```diff
@@ -94,5 +98,13 @@ void setProcessCpuTime(long processCpuTime) {
 	long getProcessCpuTime() {
 		return processCpuTime;
 	}
+	
+	/**
+         * @return cumulative CPU time in precentage (<1)
```

Comment:
What does "`<1`" here mean? Also the method is named `getCpuLoad`, so is it time or load (the description says "time", but it doesn't reflect what is returned).

Reply:
Fixed in ffdf65f8be493319c906b24c60b752782364cb35